.github/workflows/deploy.yml

```diff
@@ -0,0 +1,67 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Unit Test
```

Comment:
I think maybe we need to rename the test workflows to start with a verb like the workflows on this PR. And compass also suggests using it:
https://nimblehq.co/compass/development/code-conventions/github-actions/#naming

Reply:
Renamed in 75dcd62